### ISSUE TITLE
Update ATI destination for /scotland

### ISSUE THREAD

**Issue by jimjohnsonrollings** [bbc/simorgh#3810]
**Is your feature request related to a problem? Please describe.**
Update ATI destination for /scotland, and remove the default bucket

Arises from @sareh's comment at https://github.com/bbc/simorgh/pull/3727#discussion_r324604440

**Describe the solution you'd like**
Set the destination for /scotland as:
- 598295 (known as PLACEHOLDER) on LIVE
- 598297 (PLACEHOLDER_TEST) on TEST

Note: We will update again in future, but we need to wait for future ATI destinations to be configured.

**Testing notes**
Manual testing to make sure that the value expected (dependant on `ENV`) is returned correctly. When testing this locally, we want to create a `TEST` & `LIVE` build, run it locally and check to make sure that the correct value has been returned on the ATI Dashboard.

If you do not have login access to the ATI Dashboard speak with @jimjohnsonrollings .

Dev insight: Update unit tests.
https://github.com/bbc/simorgh/blob/latest/src/app/lib/analyticsUtils/index.js 
https://github.com/bbc/simorgh/blob/latest/src/app/lib/analyticsUtils/index.test.js 
https://github.com/bbc/simorgh/blob/latest/src/app/contexts/RequestContext/getStatsDestination/index.js

Update Cypress tests.
https://github.com/bbc/simorgh/blob/latest/cypress/support/config/envs.js
https://github.com/bbc/simorgh/blob/latest/cypress/integration/pages/testsForAllAMPPages.js
https://github.com/bbc/simorgh/blob/latest/cypress/integration/pages/testsForAllCanonicalPages.js

- [x] This feature is expected to need manual testing.

**Additional context**

**Comment by jamesdonoh** [comment]
Blocked till we know the correct destination value for /scotland.

**Comment by jimjohnsonrollings** [comment]
@jamesdonoh This is now ready to refine :)

**Comment by jimjohnsonrollings** [comment]
On hold waiting for confirmation on Thursday that we don't need to imminently change the destination again

**Comment by jimjohnsonrollings** [comment]
Unblocked
